modules/tinymce/src/core/test/ts/atomic/textpatterns/FindBlockPatternsTest.ts

```diff
@@ -27,7 +28,7 @@ describe('atomic.tinymce.textpatterns.FindBlockPatternsTest', () => {
     const defaultPatterns = patternSet.blockPatterns;
 
     const testFindStartPattern = (text: string, expectedPattern: string) => {
-      const actual = BlockPattern.findPattern(defaultPatterns, text).getOrNull();
+      const actual = BlockPattern.findPattern(sortPatterns(defaultPatterns), text).getOrNull();
```

Comment:
Instead of having to do this in every location, this would be better within `findPattern` I believe? It's also not going to add any performance overhead, in fact it'd actually reduce it as then the sorting wouldn't happen when just doing the inline key handling.